tests/Fortify/Components/DeleteUserFormTest.php

```diff
@@ -114,3 +114,17 @@
 
     $this->assertNotNull(Auth::user());
 });
+
+it('displays alert when set', function () {
+    $user = createUserModel();
+
+    $this->mock(DeleteUser::class)
```

Comment:
Not a fan of using a mock for this. Why not rely on the actual calls being executed? That way you know if a record existed before and no longer exists afterwards and you'll notice immediately when something from outside the component breaks because you didn't create your own tiny perfect world with mocks.